rafter/intro.md

```diff
@@ -0,0 +1,9 @@
+## Meet Rafter
+
+Rafter is a solution for storing and managing different types of files. It uses MinIO as object storage and gives you the ability to manage files using Kubernetes custom resources. In brief, Rafter is a fully Kubernetes-native content managing system!
```

Comment:
no no no, Rafter is not a content management system :)
please remove `In brief, Rafter is a fully Kubernetes-native content managing system!`